previews/support-policy.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Support Policy for Previews
+summary: Describes the details for the support policy for previews
+reviewed: 2020-03-26
+related:
+ - nservicebus/upgrades/support-policy
+---
+
+## Supported Versions
+
+Product Previews will always be released as a major version 0. Once a product releases version 1 or greater, it is no longer part of the Preview program and does not follow this support policy.
```

Comment:
Maybe it helps to make an example. I was wondering whether

1.0.0-alpha001 would be considered a preview until I remembered the leading zero is the difference. A small table or a few examples might make this clearer for users like

0.2.1 preview
0.15.6 preview
1.0.1 not a preview
1.0.0-alpha|beta not a preview